calico_node/filesystem/etc/calico/confd/templates/bird6.cfg.mesh.template

```diff
@@ -65,7 +67,7 @@ template bgp bgp_template {
 {{$onode_ip_key := printf "/host/%s/ip_addr_v6" .}}{{$onode_ip := getv $onode_ip_key}}
 {{$nums := split $onode_ip ":"}}{{$id := join $nums "_"}}
 # For peer {{$onode_ip_key}}
-{{if eq $onode_ip (getenv "IP6") }}# Skipping ourselves ({{getenv "IP6"}})
+{{if eq $onode_ip ($node_ip6) }}# Skipping ourselves ({{$node_ip6}})
```

Comment:
That is correct - although it is possible to change the IP address by updating the node resource... which means if you restart the node with the IP flag it'll update it back again.

I think that behavior is fine because of the change we made to the default behavior of autodetecting only if a value has not yet been assigned.